includes/Checker/Checks/Abstract_PHP_CodeSniffer_Check.php

```diff
@@ -78,17 +81,33 @@ final public function run( Check_Result $result ) {
 			);
 		}
 
+		if ( ! class_exists( '\PHP_CodeSniffer\Config' ) ) {
```

Comment:
Nit-pick:
```suggestion
		if ( ! class_exists( Config::class ) ) {
```

Reply:
FWIW it's the exact same check as for the Runner class just a few lines above. I'd need to change it for both.

Reply:
It would be cleaner to do so, rather than hard-coding the fully qualified class name again. Not critical obviously, but also there's no good reason not to :)